etc/kayobe/stackhpc-overcloud-host-images.yml

```diff
@@ -0,0 +1,28 @@
+---
+##############################
+# Release train overcloud host image sources
+
+# Whether or not to download overcloud host images from Ark
+stackhpc_download_overcloud_host_images: false
+
+# Whether or not to use images with MLNX_OFED installed (for deployment using
+# mellanox/Nvidia NICs). Only available for Ubuntu Jammy and Rocky Linux 9
+# OFED images are currently WIP and this variable is a placeholder
+stackhpc_overcloud_host_image_is_ofed: false
+
+# The overcloud host image source, defined by os_distribution, os_release,
+# stackhpc_overcloud_host_image_is_ofed, and the current stable version.
+stackhpc_overcloud_host_image_url: "{{ stackhpc_release_pulp_content_url_with_auth }}/kayobe-images/\
+                                    {{ openstack_release }}/{{ os_distribution }}/{{ os_release }}/\
+                                    {{ 'ofed/' if stackhpc_overcloud_host_image_is_ofed else '' }}\
+                                    {{ stackhpc_overcloud_host_image_version }}/\
+                                    overcloud-{{ os_distribution }}-{{ os_release }}-\
```

Comment:
trailing slash for non-OFED images?

Reply:
I don't think there would be a slash, but I have just spotted a trailing dash

Reply:
I've fixed the stray dash. Do you still think there is an issue with it somewhere?

Reply:
I meant to say dash